### PR TITLE
add maksim

### PR DESCRIPTION
**Pull request by drmowinckels**
@Mak2055  please add some information about your self and a picture. 

Focus on English, let me know when you have and I can review it and translate.

### CODE REVIEW

**Comment by Mak2055** [comment]
@drmowinckels Done!

**Comment by drmowinckels** [comment]
Where did you do changes? I cant see them in this branch :)

**Comment by Mak2055** [comment]
Ok, I really misunderstood what was expected.

I have added all you asked. My text turned out slightly longer than any of the ones that are already on the website, feel free to cut or rephrase. Also, I did not include HPC, not to overlap with Markus or Inge, you may replace 'Standardization & Optimization" with it. Generally, feel free to modify anything you think doesn't fit :D

Also, I think the cropped image should fit better with the ones you have now. I can take a new one, if you want 🤷

File: content/team/m_slivka/index.en.md [hunk 1]
```diff
@@ -0,0 +1,20 @@
+---
+title : "Maksim Slivka" # Name
+designation : 
+  - Neuroimaging data processing
+  - Optimization & standardization
+  - Machine learning
+social : # themify icon pack : https://themify.me/themify-icons or fontawesome (https://fontawesome.com/icons)
+  - icon : ti-twitter-alt
+    link : ""
```
Comment (drmowinckels) [inline content/team/m_slivka/index.en.md]:
```suggestion
```

File: content/team/m_slivka/index.en.md [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+  - icon : ti-github 
+    link : "https://github.com/Mak2055"
+  - icon: "fa-brands fa-orcid"
+    link: ""
+  - icon: "fa-brands fa-mastodon"
+    link: ""
```
Comment (drmowinckels) [inline content/team/m_slivka/index.en.md]:
```suggestion
```

File: content/team/m_slivka/index.en.md [hunk 3]
```diff
@@ -0,0 +1,20 @@
+---
+title : "Maksim Slivka" # Name
+designation : 
+  - Neuroimaging data processing
+  - Optimization & standardization
+  - Machine learning
```
Comment (drmowinckels) [inline content/team/m_slivka/index.en.md:6]:
```suggestion
  - Machine learning
  - HPC
id: m_slivka
```

File: content/team/m_slivka/index.en.md [hunk 4]
```diff
@@ -0,0 +1,20 @@
...
+    link : ""
+  - icon : ti-github 
+    link : "https://github.com/Mak2055"
+  - icon: "fa-brands fa-orcid"
+    link: ""
+  - icon: "fa-brands fa-mastodon"
+    link: ""
+---
+
+MSc in Biomedical Engineering with specialization in Computational Neuroscience, Maksim is particularly skilled in processing MR and EEG data. Being on the front end of science, he is interested in developing and applying novel solutions to scientific problems.
```
Comment (drmowinckels) [inline content/team/m_slivka/index.en.md]:
```suggestion
With a MSc in Biomedical Engineering with specialization in Computational Neuroscience, Maksim is particularly skilled in processing MR and EEG data. Being on the forefront of science, he is interested in developing and applying novel solutions to scientific problems.
```

File: content/team/m_slivka/index.no.md [hunk 5]
```diff
@@ -0,0 +1,20 @@
+---
+title : "" # Navn på person
+designation : 
+  - liste
+  - med én
+  - eller flere ferdigheter
```
Comment (drmowinckels) [inline content/team/m_slivka/index.no.md]:
```suggestion
title : "Maksim Slivka"
designation : 
  - Neuroimaging data prosessering
  - Optimalisering og standardisering
  - Maskinlæring
  - HPC
id: m_slivka
```

File: content/team/m_slivka/index.no.md [hunk 6]
```diff
@@ -0,0 +1,20 @@
...
+social :    # Kan bruke både themify (https://themify.me/themify-icons) iconer og fontawesome (https://fontawesome.com/icons)
+  - icon : ti-twitter-alt
+    link : ""
+  - icon : ti-github 
+    link : ""
+  - icon: "fa-brands fa-orcid"
+    link: ""
+  - icon: "fa-brands fa-mastodon"
+    link: ""
+gscholar_id: "" #google scholar id, for publiations list
```
Comment (drmowinckels) [inline content/team/m_slivka/index.no.md]:
```suggestion
- icon : ti-github 
    link : "https://github.com/Mak2055"
```

File: content/team/m_slivka/index.no.md [hunk 7]
```diff
@@ -0,0 +1,20 @@
...
+---
+
+Tekst om personen går her. Kort.
+Bilde i denne mappen brukes som profil bilde.
```
Comment (drmowinckels) [inline content/team/m_slivka/index.no.md]:
```suggestion
Som en Biomedisinsk ingeniør (MSc) med spesialisering i komputasonell nevrovitenskap er Maksim spesielt dyktig i prosessering av MR og EEG data. Ved æ være i spissen av forskning, er han spesielt interesert i å utvikle og applisere nye løsninger til vitenskapelige problemer.
```

**Review by drmowinckels — COMMENTED** [review]
Some suggested edits, and a Norwegian translation

**Pull request merged**